src/industry_cmd.cpp

```diff
@@ -2617,6 +2641,9 @@ static void ChangeIndustryProduction(Industry *i, bool monthly)
 	byte mul = 0;
 	int8 increment = 0;
 
+	/* Check if production changes are force-disabled */
+	if (i->ctlflags & INDCTL_NO_PRODUCTION_CHANGE) return;
```

Comment:
Suppressing the monthly callback for industries will likely break stuff badly.
It's used like "monthly cron job" for all kind of thing, not just production.

Reply:
I know it's used for all kinds of things. The intention of this flag is to allow GS to put an industry in "stasis" for when nobody is serving it, and the flag ought to be removed when any company begins serving the industry.

Reply:
Well, I would consider it rather unfair to allow GS to completely break supply mechanics in FIRS, and let users file all the reports to andy.

Reply:
How about changing this so the flag is automatically cleared if the industry transfers any cargo from or to a station?

Reply:
You may get away with ignoring the result of the callback, but not calling it at all is evil.

Reply:
Yet another alternative is changing the flag to a full-on "stasis" flag, when it's set the industry cannot produce cargo, cannot accept cargo, and does not do anything in any other manner either. (Apart from take up space on the map.) This state would probably require some new text to display in the industry window so players don't get too surprised.

Reply:
No idea why would anyone use stasis but I plan on making flags to enable setting exclusive company for industry inputs and outputs. So if that's allowed to be set as noone it would act kind of like a stasis in combination with production flags.

Reply:
"statis" sounds like a rabbit hole. Next up would be animations.